sda/treenode.go

```diff
@@ -278,20 +284,25 @@ func (n *TreeNodeInstance) dispatchHandler(msgSlice []*ProtocolMsg) error {
 	mt := msgSlice[0].MsgType
 	to := reflect.TypeOf(n.handlers[mt]).In(0)
 	f := reflect.ValueOf(n.handlers[mt])
+	var errV reflect.Value
 	if n.HasFlag(mt, AggregateMessages) {
 		msgs := reflect.MakeSlice(to, len(msgSlice), len(msgSlice))
 		for i, msg := range msgSlice {
 			msgs.Index(i).Set(n.reflectCreate(to.Elem(), msg))
 		}
 		log.Lvl4("Dispatching aggregation to", n.ServerIdentity().Address)
-		f.Call([]reflect.Value{msgs})
+		errV = f.Call([]reflect.Value{msgs})[0]
 	} else {
 		for _, msg := range msgSlice {
-			log.Lvl4("Dispatching to", n.ServerIdentity().Address)
+			log.Lvl4("Dispatching", msg, "to", n.ServerIdentity().Address)
 			m := n.reflectCreate(to, msg)
-			f.Call([]reflect.Value{m})
+			errV = f.Call([]reflect.Value{m})[0]
 		}
 	}
+	if !errV.IsNil() {
+		err := errV.Interface().(error)
+		log.Error(n.Name(), "Dispatching", mt, "yields error", err)
```

Comment:
Good point. I also moved the error-output to `dispatchMsgReader`, as this one checks the errors of both channels and handlers - and which also ignored errors ;)